client/homebrew/editor/metadataEditor/metadataEditor.less

```diff
@@ -164,7 +161,21 @@
 			.colorButton(@red);
 		}
 	}
-	.authors.field .value { line-height    : 1.5em; }
+	.authors.field {
+		.list > .tag:first-child {
+			position:relative;
+			
+			&::after {
+				content:'';
+				position:absolute;
+				border-radius:inherit;
+				inset:-1px;
+				border: 2px solid goldenrod;
+			}
+		}
+		button {color:#B00;}
```

Comment:
Use one of the colors we have defined in `colors.less` for consistency, like `redLight`.  The colors are already imported at the top of this file. 

And, if we are going to color these buttons red, we should just apply that to the broader `.tag` styling so that its consistent with the tags above.

If it were entirely up to me, I'd leave these buttons black, and flip to red on hover (with a slight easing transition, both ways), `transition: color .2s linear;`.